archeo/inc/patterns/image-with-description.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Image with description
+ */
+return array(
+	'title'      => __( 'Image with description', 'archeo' ),
+	'categories' => array( 'pages' ),
+	'content'    => '<!-- wp:columns {"align":"wide"} -->
+	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"33.33%"} -->
+	<div class="wp-block-column" style="flex-basis:33.33%"></div>
+	<!-- /wp:column -->
+	
+	<!-- wp:column {"width":"66.66%"} -->
+	<div class="wp-block-column" style="flex-basis:66.66%"><!-- wp:separator {"color":"primary","className":"is-style-wide"} -->
+	<hr class="wp-block-separator has-text-color has-background has-primary-background-color has-primary-color is-style-wide"/>
+	<!-- /wp:separator -->
+	
+	<!-- wp:paragraph -->
+	<p>' . __( 'Ancien Temple, à Chichen-Itza, applé le chateau', 'archeo' ) ) . '</p>
```

Comment:
@kjellr can we replace this with an English string?

Reply:
We've got French Strings throughout actually. They're pulled directly from the image descriptions here: 

https://digitalcollections.nypl.org/items/510d47db-11e9-a3d9-e040-e00a18064a99

At first I was into keeping them, but now that I'm thinking about it more I'm not totally sure. It seems a little weird to keep it in french. For now, I think we should just use this as is. But once all the patterns are implemented I can do a sweep through them all and replace the strings. We use the same images in both, and that way we can make sure they're all consistent.

Reply:
Created an issue for this here: https://github.com/Automattic/themes/issues/5485